examples/src/main/python/ml/pipeline_example.py

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Pipeline Example.
+"""
+from pyspark import SparkContext, SQLContext
+from pyspark.ml import Pipeline
+# $example on$
+from pyspark.ml.classification import LogisticRegression
+# $example off$
+from pyspark.ml.feature import HashingTF, Tokenizer
+from pyspark.sql import Row
+
+if __name__ == "__main__":
+
+    sc = SparkContext(appName="PipelineExample")
+    sqlContext = SQLContext(sc)
+
+    # $example on$
+    # Prepare training documents from a list of (id, text, label) tuples.
+    LabeledDocument = Row("id", "text", "label")
```

Comment:
I think we can remove the line. Because no line after here uses `LabeledDocument`. You can try to test whether the example runnable or not if we remove it.